### PR TITLE
Update minor version and add publishing task

### PR DESCRIPTION
**Pull request by MohamedSabthar**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=org.ballerinalang
-version=0.7.5-alpha2-SNAPSHOT
+version=0.8.5-alpha2-SNAPSHOT
```
Comment (ThisaruGuruge) [inline gradle.properties:3]:
```suggestion
version=0.8.0-alpha2-SNAPSHOT
```

File: tcp-ballerina/build.gradle [hunk 2]
```diff
@@ -118,7 +118,7 @@ task updateTomlFile {
     doLast {
         def stdlibDependentIoVersion = project.stdlibIoVersion.replace("${snapshotVersion}", "")
         def stdlibDependentLogVersion = project.stdlibLogVersion.replace("${snapshotVersion}", "")
-        def nettyVersion = project.nettyVersion.replace("${snapshotVersion}", "")
+        def nettyVersion = project.nettyVersion.split("-")[0]
```
Comment (ThisaruGuruge) [inline tcp-ballerina/build.gradle:121]: This is unnecessary. Netty version is already in a stable version. Why do we need to replace it anyway?

File: gradle.properties (hunk 1 above)
Comment (MohamedSabthar) [reply gradle.properties]: updated

File: tcp-ballerina/build.gradle (hunk 2 above)
Comment (MohamedSabthar) [reply tcp-ballerina/build.gradle:121]: split removed

**Review by Maninda — APPROVED** [review]

**Pull request merged**